Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py

```diff
@@ -169,7 +150,11 @@ def fetch_indicators(client: Client, indicator_type_lower: str, limit: int = -1)
                     "rawJSON": raw_data,
                     "fields": {
                         "port": argToList(item.get('tcpPorts', '')),
-                        "subfeed": item.get('serviceArea', '')
+                        "subfeed": item.get('serviceArea', ''),
+                        "office365expressroute": item.get('expressRoute', ''),
```

Comment:
We should only set the boolean values if they are returned, so don't set a default, and don't map to the indicator field _unless_ a non-`None` boolean value is returned from the `.get()`.